GW2EIEvtcParser/EncounterLogic/Raids/W1/SpiritRace.cs

```diff
@@ -177,6 +185,13 @@ internal override void ComputeNPCCombatReplayActors(NPC target, ParsedEvtcLog lo
                         replay.Trim(replay.TimeOffsets.start, hpZeroUpdate.Time);
                     }
                     break;
+                case (int)ArcDPSEnums.TrashID.WallOfGhosts:
+                    (long, long) lifespan = (target.FirstAware, target.LastAware);
+                    uint innerRadius = 400;
+                    uint outerRadius = 500;
+                    replay.Decorations.Add(new CircleDecoration(innerRadius, lifespan, Colors.Red, 0.5, new AgentConnector(target)).UsingFilled(false));
```

Comment:
Aren't this circle + the other circle just a doughnut? Or any particular reason for them to be split?